check_postgres.pl

```diff
@@ -4398,9 +4398,14 @@ sub check_custom_query {
 
         my $goodrow = 0;
 
-        ## The other column tells it the name to use as the perfdata value
+        ## If there is a single row and at least 2 columns,
+        ##  the other column tells it the name to use as the perfdata value
+        ## If there are multiple rows and at least 2 columns,
+        ##   use the second column value as the perfdata name,
+        ##   use the result value as the perfdata value.
```

Comment:
Added comments to explain exactly how the query results are processed into performance data